scripts/windows/setup/IotEdgeSecurityDaemon.ps1

```diff
@@ -534,37 +548,6 @@ function Set-Hostname {
     Write-Host "Configured device with hostname `"$Hostname`"." -ForegroundColor "Green"
 }
 
-function Set-GatewayAddress {
```

Comment:
This will break existing clients. Can you gate this behind an OS version test?

Reply:
Yes, just pushed a new commit that does this.